doc/release-notes.md

```diff
@@ -298,6 +298,9 @@ RPC
 - A new `submitheader` RPC allows submitting block headers independently
   from their block.  This is likely only useful for testing.
 
+- `getblockstats` speedup and drops `-txindex` requirement, now works on
+  all non-pruned blocks.
+
```

Comment:
nit: This should be a full sentence.

Reply:
And a separate file `./doc/release-notes-14802.md`